captum/attr/_core/integrated_gradients.py

```diff
@@ -43,14 +43,29 @@ class IntegratedGradients(GradientAttribution):
 
     """
 
-    def __init__(self, forward_func: Callable) -> None:
+    def __init__(
+        self, forward_func: Callable, use_input_marginal_effects: bool = True,
+    ) -> None:
         r"""
         Args:
 
             forward_func (callable):  The forward function of the model or any
-                          modification of it
+                    modification of it
+            use_input_marginal_effects (bool): Indicates whether to factor
```

Comment:
yes, the paper refers to the marginal effects of the input features but it looks like it can be misinterpreted. Marginal effects for continuous inputs are also less interpretable, since the unit change is being scaled. We can give it a simpler name such as: `multiply_by_inputs`.